.github/workflows/release.yml

```
@@ -142,6 +155,25 @@
           echo "binary_name=$BINARY_NAME" >> $GITHUB_OUTPUT
         id: binary
 
+      - name: Generate binary SBOM using Syft
+        shell: bash
+        run: |
+          set -euo pipefail
+          echo "Generating binary SBOM for: ${{ steps.binary.outputs.binary_name }}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)